src/npc.cpp

```diff
@@ -3350,6 +3350,16 @@ std::string npc::describe_mission() const
     } // switch (mission)
 }
 
+std::string npc::name_and_activity() const
+{
+    if( current_activity_id ) {
+        const std::string activity_name = current_activity_id.obj().verb().translated();
+        return string_format( _( "%1$s (%2$s)" ), name, activity_name );
```

Comment:
Can we add some comments for translators here?
```suggestion
        //~ %1$s - npc name, %2$s - npc current activity name.
        return string_format( _( "%1$s (%2$s)" ), name, activity_name );
```